source/quick-start.txt

```diff
@@ -10,3 +10,68 @@ Quick Start
    :depth: 2
    :class: singlecol
 
+.. include:: /includes/quick-start/overview.rst
+
+Set up Your Project
+-------------------
+
+Initialize with Go Mod
+~~~~~~~~~~~~~~~~~~~~~~
+
+Create a new directory and initialize your project with ``go mod``.
+
+.. code-block:: shell
+
+   mkdir go-quickstart
+   cd go-quickstart
+   go mod init go-quickstart
+
+.. _add-mongodb-dependency:
+
+Add MongoDB as a Dependency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use ``go get`` to add the Go driver as a dependency.
+
+.. code-block:: shell
+
+   go get go.mongodb.org/mongo-driver/mongo
+
+Create a MongoDB Cluster
+------------------------
+
+.. include:: /includes/quick-start/atlas-setup.rst
+
+
+Query Your MongoDB Cluster from Your Application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, create a file to contain your application called ``main.go``
+in the base directory of your project. Use the following sample
+code to run a query on your sample dataset in MongoDB Atlas.
+
+Set the value of the ``uri`` variable with your MongoDB Atlas connection
+string, or create an environmental variable with the name ``MONGODB_URI``
+with your Atlas connection string.
```

Comment:
Consider adding an example shell command for setting the `MONGODB_URI` variable.

E.g.
```bash
export MONGODB_URI='<connection string>'
```